io/string_writer_test.ts

```diff
@@ -33,6 +33,6 @@ Deno.test("ioStringWriterIsolationTest", async function () {
     const written = await w.write(c);
     assertEquals(written, 1);
   }
-  srcChunks[0][0] = 88;
+  // srcChunks[0][0] = 88;
```

Comment:
I think this can be deleted because the values in `srcChunks` aren't checked in the assertion